page/sass/advisors.scss

```diff
@@ -0,0 +1,47 @@
+
+.advisors p{
```

Comment:
use sass features to reduce repeated code: 
```
.advisors {
    p{ ...}
    h3 { ... }
}
```

http://sass-lang.com/guide -> nesting